scss/inc/_test-action-bars.scss

```diff
@@ -255,13 +255,11 @@
                     }
                 }
 
-                .landmark-title--hidden {
+                .landmark-title-hidden {
                     position: absolute;
-                    clip: rect(1px, 1px, 1px, 1px);
                     overflow: hidden;
                     height: 1px;
                     width: 1px;
```

Comment:
just curious why it has been hidden this way? like making a box of 1x1px

Reply:
No idea, I guess it is a deprecated code related to ie

Reply:
we need to move this class outside of &.bottom-action-bar chain, otherwise it will not be applied for the first case where is parent selector is bottom-action-bar

Reply:
Indeed, maybe we can remove it at all. It is on the main level in the other CSS file. However, I prefer to leave it there as I don't know all the elements using both CSS files.
![image](https://user-images.githubusercontent.com/11692751/135973988-913c1ff6-dc85-449f-b8e4-6a140dc79fea.png)